analyzer/src/main/kotlin/managers/CocoaPods.kt

```diff
@@ -46,7 +86,206 @@ class CocoaPods(
+        val packages = sortedSetOf<Package>()
+        val issues = mutableListOf<OrtIssue>()
+
+        if (lockfile.isFile) {
+            val dependencies = getPackageReferences(lockfile)
+
+            scopes += Scope(SCOPE_NAME, dependencies)
+            packages += scopes.flatMap { it.collectDependencies() }.map { getPackage(it, workingDir) }
+        } else {
+            issues += createAndLogIssue(
+                source = managerName,
+                message = "Missing lockfile '${lockfile.relativeTo(analysisRoot).invariantSeparatorsPath}' for " +
+                        "definition file '${definitionFile.relativeTo(analysisRoot).invariantSeparatorsPath}'. The " +
+                        "analysis of a Podfile without a lockfile is not supported at all."
+            )
+        }
+
+        val projectAnalyzerResult = ProjectAnalyzerResult(
+            packages = packages,
+            project = Project(
+                id = Identifier(
+                    type = managerName,
+                    namespace = "",
+                    name = definitionFile.relativeTo(analysisRoot).invariantSeparatorsPath,
+                    version = ""
+                ),
+                definitionFilePath = VersionControlSystem.getPathInfo(definitionFile).path,
+                authors = sortedSetOf(),
+                declaredLicenses = sortedSetOf(),
+                vcs = VcsInfo.EMPTY,
+                vcsProcessed = processProjectVcs(workingDir),
+                scopeDependencies = scopes,
+                homepageUrl = ""
+            ),
+            issues = issues
+        )
+
+        return listOf(projectAnalyzerResult)
+    }
+
+    private fun getPackage(id: Identifier, workingDir: File): Package {
+        val podspec = getPodspec(id, workingDir)
+
+        val vcs = podspec.source["git"]?.let { url ->
+            VcsInfo(
+                type = VcsType.GIT,
+                url = url,
+                revision = podspec.source["tag"].orEmpty()
+            )
+        } ?: VcsInfo.EMPTY
+
+        return Package(
+            id = id,
+            authors = sortedSetOf(),
+            declaredLicenses = listOf(podspec.license).toSortedSet(),
+            description = podspec.summary,
+            homepageUrl = podspec.homepage,
+            binaryArtifact = RemoteArtifact.EMPTY,
+            sourceArtifact = podspec.source["http"]?.let { RemoteArtifact(it, Hash.NONE) } ?: RemoteArtifact.EMPTY,
+            vcs = vcs,
+            vcsProcessed = processPackageVcs(vcs, podspec.homepage)
+        )
+    }
+
+    private fun getPodspec(id: Identifier, workingDir: File): Podspec {
+        podspecCache[id.name]?.let { return it }
+
+        val podspecName = id.name.substringBefore("/")
+
+        val podspecFile = run("spec", "which", podspecName, "--version=${id.version}", "--allow-root",
+            "--regex", workingDir = workingDir
+        ).requireSuccess().stdout.trim().let { File(it) }
```

Comment:
Nit: Formatting is a bit odd, IMO. Maybe a line break after `run(` to put all rags into a separate line?